CONTRIBUTING.md

```diff
@@ -1,30 +1,42 @@
 # Contributing to Scout
 
-Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text first. Contributing could be anything from questions and bug reports (issues), updating the documentation to fixing features.
+Hi, nice to hear that you wan't to contribute to Scout! Please have a quick read through this text 
+first. Contributing could be anything from questions and bug reports (issues), updating the 
+documentation to fixing features.
 
 ## Questions and Bugs
 
-First try to search the issues with some keywords to check if it has been touched before or if there is an ongoing discussion.
+First try to search the issues with some keywords to check if it has been touched before or if 
```

Comment:
Before submitting a new bug or question try to look into the existing issues (open or closed). Your problem could already be solved or the object of an ongoing discussion.